docker/build-tools/Dockerfile

```diff
@@ -155,6 +161,13 @@ RUN mv /tmp/linux-amd64/helm ${OUTDIR}/usr/bin
 ADD https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl ${OUTDIR}/usr/bin/kubectl
 RUN chmod 555 ${OUTDIR}/usr/bin/kubectl
 
+# Docker
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg >/tmp/dkey
+RUN apt-key add /tmp/dkey
+RUN add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu xenial stable"
+RUN apt-get update && apt-get -qqy install --no-install-recommends docker-ce="${DOCKER_VERSION}~ce~3-0~ubuntu"
```

Comment:
`-ce-3-0` should probably be in the version number, for if it changes, docker may not keep the deb in their apt repo.

Reply:
Done.